### PR TITLE
feat: add typography lab prototype

### LINKED ISSUE #399: chore: add typography lab project to frontend cert
The CSS Typography module will have a few labs for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

The previous workshop will have been the nutrition label project here
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-typography-by-building-a-nutrition-label/step-67

So, we will need a lab that covers some of the typography concepts covered from the workshop. 

## note about the HTML boilerplate

At this point in the cert, campers will have created a few labs and worked on a few workshops.
So we no longer need to force them to write out the HTML boilerplate each time.
So we don't need to create user stories for the HTML boilerplate anymore. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/css-projects` directory
- [ ] add your completed project in an `index.html` file and `styles.css` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #399

### CODE REVIEW

File: frontend-cert/css-projects/newspaper-article/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+1. You should set the root `font-size` of your HTML document to `24px`.
+2. You should have an element with a class of `newspaper` that contains all your other elements.
+3. Your `.newspaper` element should have a `font-size` of `16px` and a font of `Open Sans` with a fallback font of `sans-serif`.
+4. Within your `.newspaper` element, you should have at least seven more elements. One for the newspaper name that has a class of `name`, one for the the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`.
+5. Your `.name` element should have a `font-size` that's two times the size of the root elements `font-size` and a font of `Times New Roman` with a fallback font of `serif`.
+6. Your `.name` and `.author` elements should use CSS to make all its characters uppercase.
```
Comment (zairahira) [inline frontend-cert/css-projects/newspaper-article/user-stories.md:6]: I couldn't figure out if `text-transform` has been taught at this point.
I couldn't find it in the nutrition label.

Here are the lecture contents according to the sheet:

- Why should you use relative units for font size?
- What are web safe fonts?
- Why is it best practice to include a fallback value for font-family?
- What is the @font-face at rule and how does it work?
- How to work with external fonts like Font Squirrel and Google Fonts.
- How multiple fonts can increase page download size.

File: frontend-cert/css-projects/newspaper-article/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+7. Your `.headline` element should `font-size` that's two times the size of its parent's `font-size` and use CSS to make it bold.
+8. Your `.sub-headline` element should have a `font-weight` of `100`, a `font-size` that's `1.5` times the size of its parent's `font-size`, and use CSS to make it italicized.
+9. Your `.author` should use CSS to make it bold.
+10. Your `.text` elements should be indented `20px`.
+11. Your `.text` elements should have a `line-height` that's two times the size of its parent's `font-size`.
+12. The first letter of your `.text` elements should be bold and two times the size its parent's `font-size`. Hint: use the `::first-letter` selector.
```
Comment (zairahira) [inline frontend-cert/css-projects/newspaper-article/user-stories.md:12]:
```suggestion
12. The first letter of your `.text` elements should be bold and two times the size of its parent's `font-size`. Hint: use the `::first-letter` selector.
```

File: frontend-cert/css-projects/newspaper-article/user-stories.md (hunk 1 above)
Comment (moT01) [reply frontend-cert/css-projects/newspaper-article/user-stories.md:6]: I think it should be fine. It looks like it's covered in the flexbox workshop, which is right before the nutrition label.

**Review by zairahira — APPROVED** [review]

**Review by Dario-DC — APPROVED** [review]
This is a fun project :alien:

**Pull request merged**